bktest/cli.py

```diff
@@ -23,14 +23,18 @@
 
 
 @click.group(invoke_without_command=True)
+#
 @click.option('--start', type=click.DateTime(formats=["%Y-%m-%d"]), default=None, help="Start date.")
 @click.option('--end', type=click.DateTime(formats=["%Y-%m-%d"]), default=None, help="End date.")
-@click.option('--offset-before-trading', type=int, default=1, show_default=True, help="Number of day to offset to push the signal before trading it.")
+#
+@click.option('--offset-before-trading', type=int, default=0, show_default=True, help="Number of day to offset to push the signal before trading it.")
```

Comment:
"Number of day to offset to push the signal before trading it."
->
"Number of day to offset to push each date of the portfolio before trading it."

Reply:
Fixed
https://github.com/crunchdao/backtest/pull/29/commits/104c005a21f871fd30bddb4235a3195200ab445e